services/test_results.py

```diff
@@ -246,38 +221,6 @@ def build_message(self, payload: TestResultsNotificationPayload) -> str:
 
         return "\n".join(message)
 
-    async def notify(self, payload: TestResultsNotificationPayload) -> Tuple[bool, str]:
-        self.repo_service = get_repo_provider_service(self.commit.repository)
-
-        await self.get_pull()
-        if self.pull is None:
-            log.info(
-                "Not notifying since there is no pull request associated with this commit",
-                extra=dict(
-                    commitid=self.commit.commitid,
-                ),
-            )
-            return False, "no_pull"
-
-        message = self.build_message(payload)
-
-        sent_to_provider = await self.send_to_provider(message)
-        if sent_to_provider == False:
-            return (False, "torngit_error")
-
-        return (True, "comment_posted")
-
-    def insert_breaks(self, table_value):
```

Comment:
was this in the code but not being used? 💀

Reply:
yes 😅